apps/prairielearn/src/pages/instructorAssessmentRegrading/instructorAssessmentRegrading.html.ts

```diff
@@ -0,0 +1,159 @@
+                  csrfToken: resLocals.__csrf_token,
+                })}
+              `
+            : ''}
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">
+              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Regrading
+            </div>
+
+            ${resLocals.authz_data.has_course_instance_permission_edit
+              ? html`
+                  <div class="card-body">
+                    <p>
+                      <button
+                        type="button"
+                        class="btn btn-primary"
+                        data-toggle="modal"
+                        data-target="#regrade-all-form"
+                      >
+                        <i class="fa fa-sync" aria-hidden="true"></i>
+                        Regrade all assessment instances
+                      </button>
+                    </p>
+                    <p class="small">
+                      To regrade assessment instances for specific students, use the "Action"
+                      dropdown on the
+                      <a
+                        href="${resLocals.urlPrefix}/assessment/${resLocals.assessment
+                          .id}/instances"
+                        >assessment instances page </a
+                      >.
+                    </p>
+                  </div>
+                `
+              : ''}
+
+            <div class="table-responsive">
+              <table class="table table-sm table-hover">
+                <thead>
+                  <tr>
+                    <th>Number</th>
+                    <th>Date</th>
+                    <th>Description</th>
+                    <th>User</th>
+                    <th>Status</th>
+                    <th>Actions</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  ${regradingJobSequences && regradingJobSequences.length > 0
+                    ? regradingJobSequences.map((jobSequence) => {
+                        return html`
+                          <tr>
+                            <td>${jobSequence.job_sequence.number}</td>
+                            <td>${jobSequence.start_date_formatted}</td>
+                            <td>${jobSequence.job_sequence.description}</td>
+                            <td>${jobSequence.user_uid}</td>
+                            <td>
+                              ${renderEjs(__filename, "<%- include('../partials/jobStatus'); %>", {
+                                status: jobSequence.job_sequence.status,
+                              })}
+                            </td>
+                            <td>
+                              <a
+                                href="${resLocals.urlPrefix}/jobSequence/${jobSequence.job_sequence
+                                  .id}"
+                                class="btn btn-xs btn-info"
+                                >Details</a
```

Comment:
Add spaces to make Prettier work its magic here.

Reply:
Added spaces here in [7ce1e03](https://github.com/PrairieLearn/PrairieLearn/pull/9772/commits/7ce1e035111dff8e1e3797d6d27226af666f10bc).